### PR TITLE
Updated meta data and links for tamil news channels

### PR DESCRIPTION
**Pull request by javithakthar**

### CODE REVIEW

**Review by freearhey — DISMISSED** [review]

File: channels/in.m3u [hunk 1]
```diff
@@ -546,8 +556,8 @@ http://rtmp.santhoratv.zecast.net/santhoratv/santhoratv/playlist.m3u8
 http://santhoratv.zecast.net/santhoratv/santhoratv/index.m3u8
 #EXTINF:-1 tvg-id="SardariTV.in" tvg-country="IN" tvg-language="Hindi" tvg-logo="https://i.imgur.com/6Q844bs.jpg" group-title="",Sardari TV (1080p)
 http://158.69.124.9:1935/sardaritv/sardaritv/playlist.m3u8
-#EXTINF:-1 tvg-id="SathiyamTV.in" tvg-country="IN" tvg-language="Tamil" tvg-logo="" group-title="",Sathiyam TV (1080p) [Not 24/7]
-https://iptv.dc-team.com/query?url=https://www.youtube.com/channel/UC2ziCMHFPWkFHjocUMXT__Q/live
+#EXTINF:-1 tvg-id="SathiyamTV.in" tvg-country="IN" tvg-language="Tamil" tvg-logo="https://static.wikia.nocookie.net/logopedia/images/c/c9/SathiyamFinallogo.png/revision/latest/scale-to-width-down/180?cb=20191225174913" group-title="News",Sathiyam TV (1080p)
+https://iptv.dc-team.com/query?url=https://www.youtube.com/watch?v=zt5J7ZQMZ-A
```
Comment (jkemp814) [inline channels/in.m3u:560]: Could not get the link, Streamlink couldn't read https://www.youtube.com/watch?v=zt5J7ZQMZ-A

Direct from Youtube:
Private Video
Sign in if you've been granted access to this video

**Review by jkemp814 — COMMENTED** [review]
All play except this one

File: channels/in.m3u (hunk 1 above)
Comment (javithakthar) [reply channels/in.m3u]: Reverted the URL back to older one. Older URL works. Thanks for notifying me!!!

**Comment by javithakthar** [comment]
I've resolved review comments and merge conflict. Kindly check @jkemp814 @freearhey when you've time.

File: channels/in.m3u [hunk 2]
```diff
@@ -557,7 +557,7 @@ http://santhoratv.zecast.net/santhoratv/santhoratv/index.m3u8
 #EXTINF:-1 tvg-id="SardariTV.in" tvg-country="IN" tvg-language="Hindi" tvg-logo="https://i.imgur.com/6Q844bs.jpg" group-title="",Sardari TV (1080p)
 http://158.69.124.9:1935/sardaritv/sardaritv/playlist.m3u8
 #EXTINF:-1 tvg-id="SathiyamTV.in" tvg-country="IN" tvg-language="Tamil" tvg-logo="https://static.wikia.nocookie.net/logopedia/images/c/c9/SathiyamFinallogo.png/revision/latest/scale-to-width-down/180?cb=20191225174913" group-title="News",Sathiyam TV (1080p)
-https://iptv.dc-team.com/query?url=https://www.youtube.com/watch?v=zt5J7ZQMZ-A
+https://iptv.dc-team.com/query?url=https://www.youtube.com/channel/UC2ziCMHFPWkFHjocUMXT__Q/live
```
Comment (jkemp814) [inline channels/in.m3u:560]: Works

**Review by jkemp814 — APPROVED** [review]

**Review by freearhey — APPROVED** [review]

**Pull request merged**